android/beagle/src/main/java/br/com/zup/beagle/android/view/custom/BeagleNavigator.kt

```diff
@@ -86,7 +86,8 @@ internal object BeagleNavigator {
 
     fun popToView(context: Context, route: String) {
         if (context is AppCompatActivity) {
-            context.supportFragmentManager.popBackStack(route, 0)
+            val relativePath = route.replace(BeagleEnvironment.beagleSdk.config.baseUrl, "")
```

Comment:
What do you think of use `removePrefix` method instead?
```suggestion
            val relativePath = route.removePrefix(BeagleEnvironment.beagleSdk.config.baseUrl)
```